.github/workflows/ci.yml

```diff
@@ -1,34 +1,42 @@
 name: Lint and Test Charts
 
-on:
-  pull_request:
-    paths:
-      - 'charts/**'
+on: pull_request
```

Comment:
I think this change interests me the most of this whole PR.

Reply:
> It's a bit odd having both changes to charts and changes to the github action in the same PR...

This PR is a messy wip, I'm going to clean things up and do the ci separately if I can get it to work but for now I'm just using this PR to test the ci because ci will only run fully if it detects chart changes (or at least it used to). I may ditch ci temporarily.. I'll ask for your review again when things are cleaned up (sorry for all the notifs)

Reply:
So, you might try [nektos/act](https://github.com/nektos/act)

Reply:
I'm converting this PR to a draft (we should actually use Drafts more eagerly to explain to people that a given PR isn't ready to be merged).